### DISCUSSION TITLE
What is the PRO label in my profile?

### DISCUSSION THREAD

**Discussion by ygnoh** [community/community#23345] [New to GitHub]
<p>Today I visited Github after a long interval. I found the PRO lable in my profile:</p>
<p><img src="https://user-images.githubusercontent.com/13075245/181076417-dcd6e117-6602-4804-aebf-d2d4c4e38b7a.png" alt="스크린샷 2019-01-08 오후 3.17.25.png" data-base62-sha1="Proa4sxm78pWSc56l21INutiKi" width="526" height="470"></p>
<p>What is that and What does that mean?</p>
<p>Thanks.</p>

**Comment by mpboom** [comment, marked as answer]
<p>It means you are subcribed to the new Pro plan, as explained in this blog post: <a href="https://blog.github.com/2019-01-07-new-year-new-github/" rel="nofollow noopener">https://blog.github.com/2019-01-07-new-year-new-github/</a></p>

**Comment by necaris** [comment]
<p>For those of us who have been using the “Developer” tier for the unlimited private repositories, it’s really unclear what the benefit of continuing with the “Pro” plan is. Can you clarify?</p>
<p>Note: I see that it says “advanced code review tools” in the pricing page, but without a concrete example that doesn’t mean much to me, so I’d appreciate any more details.</p>

**Comment by mpboom** [comment]
<p>The extra features you get are indeed listed right here: <a href="https://github.com/pricing" rel="nofollow noopener">https://github.com/pricing</a></p>
<p>If you think you don’t need the extra features, you can always downgrade your account to a free one.</p>
<p>About the advanced code review tools; I think it would be handy if a mod jumped in and clarified that.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/necaris">@necaris</a>, Thanks for being here! Advanced code review tools, include Codeowners and Protected Branches.</p>

**Comment by rahilm97** [comment]
<p>I signed up for a student account for access to free private repositories. I see Pro on my profile, should I now downgrade to Free?</p>

**Comment by mpboom** [comment]
<p>On <a href="https://github.com/pricing" rel="nofollow noopener">https://github.com/pricing</a> you can compare the different plans. If you think you don’t need Pro anymore you are absolutely free to downgrade at anytime.</p>

**Comment by ocano-diego** [comment]
<p>Thought it was something useful and relevant to the comunity but nah is just ads. Why would I like to tell everybody what plan I am using, I find this very intrusive.</p>

**Comment by mpboom** [comment]
<p>Hi <a class="mention" href="/poncianodiego">@poncianodiego</a>,</p>
<p>You can disable the badge at any time in your profile settings.</p>

**Comment by kcroker** [comment]
<p>Thank you for the instructions on how to disable the badge.</p>

**Comment by ghost** [comment]
<p>if your using a school email for the pro account I reccomend making a private account to transfer your repos over before you graduate. this way you dont loose your repo’s if the school kills your email. Im not entirely sure how it would work but thats something I would do to stay on the safe side. You should keep you PRO account so you can create private repositories for school work (avoid having to deal with other students stealing your code)</p>

**Comment by mpboom** [comment]
<p><a class="mention" href="/dixantpatel">@dixantpatel</a> private repo’s are availible on Free accounts as well since a few months ago. You can always update the main mail address of your account to a mail address you fully control, without having to transfer everything over to another account.</p>

**Comment by earegun** [comment]
<p>Actually, those who use the GitHub pro plan only, they are able to get Pro Label :blush:</p>

**Comment by bydevmar** [comment]
The **"Pro"** tag on GitHub profiles signifies that the user is a **GitHub Pro** subscriber. Let me explain what this means:

1. **GitHub Pro**:
   - **GitHub Pro** is a subscription plan offered by GitHub. It provides additional features and benefits beyond what's available in the free version.
   - With **GitHub Pro**, developers can better manage their projects, collaborate with others, and enhance their overall GitHub experience.
   - Here are some key features of **GitHub Pro**:
     - **Unlimited Private Repositories**: GitHub Pro users can create an unlimited number of private repositories. Previously, free accounts were limited to public repositories only.
     - **Advanced Code Review Tools**: Features like **Codeowners** and **Protected Branches** are available to help maintain code quality and security.
     - **Collaboration**: While free accounts allow up to three collaborators per private repository, **GitHub Pro** users can collaborate with more team members.
     - **Personalization**: **Badges and Achievements** highlight your activity and show if you use **GitHub Pro** or participate in other programs like the **Arctic Code Vault**, **GitHub Sponsors**, or the **GitHub Developer Program**³.

2. **GitHub Developer Program**:
   - The **GitHub Developer Program** is a program for developers who integrate with the GitHub API in their applications.
   - Anyone can register for this program and receive a badge. To register, you need to provide your email and a URL (such as a link to your Medium profile) during the registration process².

In summary, having the **"Pro"** tag on your GitHub profile indicates that you are a **GitHub Pro** subscriber, enjoying additional features and benefits to enhance your development workflow. 🚀.

**Comment by NoahRoters** [comment]
Hello, I just noticed that my account has the "Pro" tag.

However, I don’t know since when I have the Pro subscription, and I didn’t subscribe to it myself. I also didn’t do anything special recently. I only created a few new repositories, forked a repository, was invited to a private repository to collaborate on a school project, and added my school email to my account.

My question is whether I might receive an invoice or run into any issues because of this, or if everything is normal and fine as it is.

GitHub is working fine for me right now. I’m also not completely sure if I actually need the Pro features, but I can’t see where to manage or edit the subscription in my account. On the pricing page, nothing is shown for me.

Is it okay if I just leave it as it is, or do I need to change anything?

Thanks in advance for the help and sory for my bad englisch :)

**Reply by Codrax** [reply]
I have noticed the same thing on my profile, and on the profile of a few people I know which  have not subscribed to the GitHub Pro plan. This may be just a visual bug or maybe the label is automatically assigned once certain conditions are met and never revoked?

In any case I don't believe it's anything to cause concern.
